### PR TITLE
added remove feature

### PR DESCRIPTION
**Pull request by kmisterios**
Check out new feature :)

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
     pass
 
+
 def test_remove():
-    pass
+    st = Storage({'a': 1, 'b': 2})
+    key = 'b'
+    val = st.remove(key)
+    assert val == 2, "Key {} is not in the dictionary".format(key)
+    key = 'c'
```
Comment (maxkuzn) [inline tests.py]: Are you sure you wanted to check a 'c' key, not a removed 'b' key?

File: tests.py [hunk 2]
```diff
@@ -1,14 +1,24 @@
 from storage import Storage
 
+
 def test_add():
     pass
 
+
 def test_remove():
```
Comment (maxkuzn) [inline tests.py:31]: Add a test on removing an unexisting key.

**Review by maxkuzn — CHANGES_REQUESTED** [review]

File: tests.py (hunk 1 above)
Comment (kmisterios) [reply tests.py]: Ok

File: tests.py [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
-    pass
+    st = Storage({'a': 1, 'b': 2})
+    key = 'b'
+    val = st.remove(key)
```
Comment (maxkuzn) [inline tests.py:34]: Add check that the key is really removed. So get should return None.

**Review by maxkuzn — CHANGES_REQUESTED** [review]

**Review by maxkuzn — APPROVED** [review]

**Pull request merged**